### PR TITLE
Heart Bug Fix

### PR DESCRIPTION
**Pull request by zachary-nguyen**
#### Information Heart Bug Fix
 - Kind: Bug Fix
 - Emoji that best describe this PR: <3

#### Description 

- Added static from method to heart class.
- Fixed heart radius to be representative of actual radius
- Centered heart
- updated test harness
#### PR checklist

If you've made change to the code, be sure to validate this checklist:

 - [x] I have read the [contribution guideline](../contributing.md)
 - [x] I have minimized linting error when running `npm run lint`
 - [x] I have added tests for my code
 - [x] I have successfully ran tests with `npm test`
 - [x] I have checked that everything looks good on [drawing tests](../drawingTests/index.html)
 - [x] I am an awesome human being

### CODE REVIEW

File: modules/pencil.js/readme.md [hunk 1]
```diff
@@ -51,18 +51,16 @@ const scene = new Pencil.scene();
 // Just the part you need
 import Scene from "@pencil.js/scene";
 // or
-import { Scene } from "pencil.js";
+import Scene from "pencil.js";
```
Comment (zachary-nguyen) [inline modules/pencil.js/readme.md]: I'm not sure if this was done by me or from pulling upstream I can revert the import changes if need be

**Comment by GMartigny** [comment]
To do things "by the books" you'll need to:
 - Add the main repo as reference
	`git add remote upstream https://github.com/pencil-js/pencil.js.git`
	`git fetch upstream`
 - Revert your master branch to its original state while keeping your changes
	`git revert upstream/master --soft`
 - Update your master branch
	`git merge upstream/master`
 - Create another branch for your changes
	`git checkout -b heartBugFix`
 - Commit and push your changes to this branch
 - Create a PR using this branch

I know git can be a bit weird some times, if you need some more guidance you can [ping me in Discord](https://discord.gg/GkEgjsy).
Otherwise, if you can't or don't want to, I can do with your PR as is (don't worry to much).

File: modules/pencil.js/readme.md (hunk 1 above)
Comment (GMartigny) [reply modules/pencil.js/readme.md]: This is an older version of the code from upstream. Trying merging first.

**Comment by zachary-nguyen** [comment]
I just manually changed the stuff and repushed from my master. I should've been working from a separate branch instead, oops!

**Comment by GMartigny** [comment]
I move thing around while keeping your values to simplify the code.
Once again, thanks a lot for working on that, I love the result.
Fix #62

**Pull request merged**